js/index.js

```diff
@@ -38,5 +40,94 @@ const siteContent = {
 };
 
 // Example: Update the img src for the logo
+
+//nav selector
+const [ nav ] = document.getElementsByTagName('nav');
```

Comment:
Yaaas, destructure that array, hunny.

Feel free to check out this [organised list of ES6 features](http://es6-features.org) for more nifty little tricks. Learn these, and you'll be miles ahead of other developers.